docs/Sprint-Review/sprint-92-summary.md

```diff
@@ -0,0 +1,62 @@
+# Sprint 92 Summary
+
+01/31/2024 - 02/13/2024
+
+Velocity (Dev): 6
+
+## Sprint Goal
+* Dev:
+    * Continue parsing engine development and begin work on enhancement tickets
+        * #2536 Cat 4 validation 
+    * #1858 Secure OFA staff access to Kibana 
+        * Unblocks #1350 when complete 
+* DevOps:
+    * #2790 - Update deployment code to support Kibana and integrate with Standing Elastic instance
+* Design: 
+    * Tie up current documentation work
+    * Continue refinement of research roadmap
+
+
+## Tickets
+### Completed/Merged
+* [#2751 Resource Card updated with latest coding instructions](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2751)
+
+### Ready to Merge
```

Comment:
@reitermb i didnt review any of this until sprint 93.

Reply:
@ADPennington Sorry, I think I got something jumbled up in the re-handoff from Connor. Just to make sure I get any changes down right...

Based on what I'm seeing in GitHub:
- #2772 should be removed from this summary as it was closed on Jan 18
- #1858 / #2775 should stay but be bumped up to merged as it was merged on Feb 12
- #1350 should be removed, still in progress/review 
- #2711 should be removed as it was merged back on Jan 22

Does that sound right?

Reply:
thank you for checking, @reitermb  thats correct.